packages/react-components/src/Concepts/Positioning/PositioningShorthandPositions.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { PositionedComponent, useGridExampleStyles } from './utils.stories';
+
+export { Default } from './PositioningDefault.stories';
+
+export const ShorthandPositions = () => {
+  const styles = useGridExampleStyles();
+
+  return (
+    <>
+      <div className={styles.targetContainer}>
+        <div className={styles.instructions}>Hover over each box to see its positioned element</div>
+        <PositionedComponent
+          positioning="above-start"
+          targetClassName={styles.aboveStart}
+          targetContent="above-start ↑"
+        />
+        <PositionedComponent positioning="above" targetClassName={styles.above} targetContent="above ↑" />
+        <PositionedComponent positioning="above-end" targetClassName={styles.aboveEnd} targetContent="above-end ↑" />
+
+        <PositionedComponent positioning="before-top" targetClassName={styles.beforeTop} targetContent="← before-top" />
+        <PositionedComponent positioning="before" targetClassName={styles.before} targetContent="← before" />
+        <PositionedComponent
+          positioning="before-bottom"
+          targetClassName={styles.beforeBottom}
+          targetContent="← before-bottom"
+        />
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/132231561-90ebcc9e-6f00-4fc2-b831-351e70f049af.png)

What is we will make these square? Arrows then will positions better

Reply:
Larger buttons really doesn't work, and I prefer this layout over the Popper one which is less verbose https://popper.js.org/

I've made the popover larger so that the differences are clearer, that should be enough

Reply:
Popper's layout doesn't really work for `coverTarget` 

I've made the popover larger in the examples so the differences are clearer. That should be enough

Reply:
OK I tried to do smth with flex and css let me know what you think